commands/releases/output.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+let cli = require('heroku-cli-util')
+let co = require('co')
+let request = require('request')
```

Comment:
I'm not seeing a lot of documentation for using heroku-cli-util :(
This seems to be working though: d33869b4866bbf049e47bcac499bf55730d7be0c